File: leetcode_119/solution.py
class Solution:
    def getRow(self, rowIndex):
        """
        :type rowIndex: int
        :rtype: List[int]
        """
        ret = [1 for _ in range(rowIndex + 1)]
        for i in range(2, rowIndex + 1):
            for j in range(i - 1, 0, -1):
                ret[j] += ret[j - 1]
        return ret


if __name__ == '__main__':
    print(Solution().getRow(5))

File: leetcode_119/solution.go
package main

import "fmt"

func getRow(rowIndex int) []int {
	row := make([]int, rowIndex + 1)
	row[0] = 1

	for i := 1; i < rowIndex + 1; i++ {
		for j := i; j > 0; j-- {
			row[j] += row[j - 1]
		}
	}

	return row
}

func main() {
	rowIndex := 4
	fmt.Println(getRow(rowIndex))
}
